bin/crew

```diff
@@ -175,6 +175,8 @@ at_exit do
   GnomePostinstall.run unless GnomePostinstall.gnome_packages.blank?
   # Print exit messages.
   ExitMessage.print
+  # Replace the currently running Ruby process with a new instance of the parent shell (thus sourcing the relevant configuration)
+  exec(`ps -p #{Process.ppid} -o comm=`.chomp)
```

Comment:
I highly doubt this works the way you think.  Sourcing has to be done in the current working session or it won't work.

Reply:
It works. I've tested it.